src/graph_world/nodeclassification/generator_wrappers.py

```diff
@@ -79,4 +81,44 @@ def Generate(self, sample_id):
                 graph_memberships=sbm_data.graph_memberships,
                 node_features=sbm_data.node_features,
                 feature_memberships=sbm_data.feature_memberships,
-                edge_features=sbm_data.edge_features)}
\ No newline at end of file
+                edge_features=sbm_data.edge_features)}
+
+@gin.configurable
+class CABAMGeneratorWrapper(GeneratorConfigSampler):
+
+  def __init__(self, param_sampler_specs, marginal=False,
+               normalize_features=True):
+    super(CABAMGeneratorWrapper, self).__init__(param_sampler_specs)
+    self._marginal = marginal
+    self._normalize_features = normalize_features
+    self._AddSamplerFn('nvertex', self._SampleUniformInteger)
+    self._AddSamplerFn('m', self._SampleUniformInteger)
+    self._AddSamplerFn('assortativity_type', self._SampleUniformInteger)
+    self._AddSamplerFn('c_probs_in', self._SampleUniformFloat)
+    self._AddSamplerFn('c_probs_out', self._SampleUniformFloat)
+
+
+  def Generate(self, sample_id):
```

Comment:
Is it true that assortativity type currently plays no role? I don't see it in the inputs/outputs